.github/workflows/workflow.yaml

```diff
@@ -153,7 +153,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: superfly/flyctl-actions/setup-flyctl@67ddda2a87628f1aff28a54b8a708879ec62c80f
+      - uses: superfly/flyctl-actions/setup-flyctl@adbb124b95f2e47566253d95497432c27f4bdd70
```

Comment:
This is wrong. https://github.com/superfly/flyctl-actions/commit/67ddda2a87628f1aff28a54b8a708879ec62c80f is the latest commit. https://github.com/superfly/flyctl-actions/commit/adbb124b95f2e47566253d95497432c27f4bdd70 is from a month and a half before that. I think they just created the release to correspond with the 1.3 tag.